client/src/utilities/api-calls.js

```diff
@@ -1,6 +1,6 @@
 const fetchCall = (entry, id) => {
   const method = entry && id ? "PUT" : entry ? "POST" : "GET";
-  const body = entry?.date ? entry : entry ? {journal_entry: entry} : null;
+  const body = entry?.date ? entry : entry && {journal_entry: entry};
```

Comment:
YES! Love this!! The && operator consistently slips my mind, so thank you for snagging this!